Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -605,6 +628,8 @@ instance : AddCommMonoid (LieSubmodule R L M) where
   add_comm := sup_comm
   nsmul := nsmulRec
 
+variable (N N')
```

Comment:
I don't understand why this is needed ? [LieSubmodule.add_eq_sup](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Lie/Submodule.html#LieSubmodule.add_eq_sup) already has `N` and `N'` explicit.

Reply:
:facepalm: